config/config.default.yaml

```diff
@@ -105,6 +105,8 @@ co2_budget:
   2045: 0.050
   2050: 0.000 # climate-neutral by 2050
 
+ember_hotfix: true #implement temporary fixes for the project, to be implemented within the workflow when completely approved.
+
```

Comment:
not the right location, unclear name of the setting

I'd suggesting putting it under ember_settings and renaming it to remove_stores